lib/bundlex/toolchain/visual_studio.ex

```diff
@@ -16,10 +16,10 @@ defmodule Bundlex.Toolchain.VisualStudio do
   alias Bundlex.Native
   alias Bundlex.Output
 
-  @directory_wildcard_x64 "c:\\Program Files (x86)\\Microsoft Visual Studio *"
-  @directory_wildcard_x86 "c:\\Program Files\\Microsoft Visual Studio *"
-  @directory_env "VISUAL_STUDIO_ROOT"
+  @program_files System.get_env("ProgramFiles(x86)") |> Path.expand()
```

Comment:
```suggestion
  @program_files System.fetch_env!("ProgramFiles(x86)") |> Path.expand()
```

also, when this directory doesn't exist, Bundlex fails to compile, see https://app.circleci.com/pipelines/github/membraneframework/bundlex/463/workflows/326dfb79-55db-43a6-8db3-e09e5f878e8a/jobs/1015

So I'd make this `defp`

Reply:
@harrisi please change to defp too ;)

Reply:
Sorry, for some reason I wanted to try the GitHub-provided merge tool on mobile while in bed last night. Didn't go as planned!

Anyway, I realized these are only used in one place anyway, so I just put them where they're used to consolidate it.